lib/src/models/sent_file_info.dart

```diff
@@ -0,0 +1,8 @@
+
+/// Implementation independent information on sent file.
+class SentFileInfo {
```

Comment:
This class has an odd name and feels unnecessary. I also noticed that it's not exported.